### DISCUSSION TITLE
Is it possible to disable events when im pressing on a chart?

### DISCUSSION THREAD

**Discussion by RafaelCENG** [wuba/react-native-echarts#77] [Q&A] [answered]
First of all, thank you for your hard work. My question is, is it possible to disable the events that occur when I press on a chart? For example, I currently have a pie chart where, upon pressing on a section, a label appears with its corresponding value. However, when I drag my finger, it allows me to select other parts of the pie chart. The problem is that I have a tab system or scroll view, and when I press the pie chart and drag my finger, it enables scrolling or switches tabs. Can you assist me with this issue?

Thanks in Advance. If you need a video to understand it better i can create one.

**Comment by zhiqingchen** [comment, marked as answer]
it works if you know how to custom react-native-gesture-handler.
here is the document https://wuba.github.io/react-native-echarts/docs/advanced-guides/use-rngh#custom-gestures

a example here https://wuba.github.io/react-native-echarts/docs/expo-snacks/custom-gesture-use-rngh

**Comment by zhiqingchen** [comment]
In short, you can activate the relevant gesture after a certain delay of long press.

```js
defaultGestures[0].activateAfterLongPress(250);
```

**Reply by RafaelCENG** [reply]
Thanks will test it.

**Comment by zhiqingchen** [comment]
If you don't need any envents at all, you can consider disabling it directly.
```
<SvgChart ref={chartRef} handleGesture={false} />;
```
